media/js/firefox/nothing-personal/animate-pop-in.es6.js

```diff
@@ -0,0 +1,103 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+let observer;
+let browser;
+let attachedStickyNote;
+let detachedStickyNote;
+let stickyGif;
+
+function createObserver() {
+    return new IntersectionObserver(function (entries) {
```

Comment:
**praise:** love seeing intersection observer used for performant scroll animations 🔥